build.gradle

```diff
@@ -25,15 +25,23 @@ subprojects {
     // source-code version and must be <= targetCompatibility
     targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
-    def junit5Version = '5.6.0'
+    ext{
+        jacksonVersion = '2.10.2'
```

Comment:
we cannot define global variables but we can define extra properties for libraries that are used in multiple projects